src/lucky/page_helpers/time_helpers.cr

```diff
@@ -35,7 +35,7 @@ module Lucky::TimeHelpers
 
   private def distance_in_minutes(distance : Int32) : String
     case distance
-    when      1 then "a minute"
+    when 1      then "a minute"
     when 2...45 then "#{distance} minutes"
     else
       "about an hour"
```

Comment:
Is this correct? Could `distance` be more than 60 minutes?

Reply:
Good question! It’s inky when less than 60: 
https://github.com/luckyframework/lucky/blob/0e6ee45d851ccc872ae06882a11ee354d0af37e2/src/lucky/page_helpers/time_helpers.cr#L10